lib/engine/game/g_18_tokaido/meta.rb

```diff
@@ -26,12 +26,12 @@ module Meta
           {
             sym: :newbie_rules,
             short_name: 'Local 普通',
-            desc: '[newbie variant] slightly expanded tileset, removes yellow zone from stock market',
+            desc: '[Beginner variant] expanded tileset and removes yellow zone from stock market',
           },
           {
             sym: :limited_express,
             short_name: 'Limited Express 特急',
-            desc: '[brutal variant] one less 6 train, 50% to float, 1882-like stock market',
+            desc: '[Experienced variant] one fewer 6 train, only 50% to float, 1882-like stock market',
```

Comment:
I'd quibble.  This isn't really an "experienced" variant, it's just brutal.  There's a difference.